docs/guides/core-concepts/introduction-to-cypress.mdx

```diff
@@ -1096,17 +1080,16 @@ fail! Here's a few:
 
 <strong>Core Concept</strong>
 
-With Cypress, you don't have to assert to have a useful test. Even without
-assertions, a few lines of Cypress can ensure thousands of lines of code are
-working properly across the client and server!
+With Cypress, you don't have to write explicit assertions to have a useful test.
+Without a single `expect()` or `.should()`, a few lines of Cypress can ensure
+thousands of lines of code are working properly across the client and server.
 
-This is because many commands have a built in
-[Default Assertion](#Default-Assertions) which offer you a high level of
-guarantee.
+This is because many commands have a built in Implicit Assertions which offer
```

Comment:
```suggestion
This is because many commands have built in Implicit Assertions which offer
```

Reply:
Fixed in [`2929a65` (#5315)](https://github.com/cypress-io/cypress-documentation/pull/5315/commits/2929a652fcbfe77ed7236ab2f2f306b1b8e297ea)